### PR TITLE
[PLAYER-5544] App gets crashed when Clicked on the “Previous/Next Video” icon.

### PR DESCRIPTION
**Pull request by ipadilla4**
This fix consists in implement "onSwitch" function in React bridge, which was causing crashes because it wasn't implemented.

### CODE REVIEW

**Review by cacel — APPROVED** [review]
LGTM

File: sdk/iOS/OoyalaSkinSDK/React/OOReactSkinBridgeModuleMain.m [hunk 1]
```diff
@@ -146,6 +146,16 @@ - (void)setSkinViewDelegate:(nonnull id<OOReactSkinModelDelegate>)delegate {
   }
 }
 
+RCT_EXPORT_METHOD(onSwitch:(NSDictionary *)params) {
+  NSMutableDictionary *result = [[NSMutableDictionary alloc] initWithDictionary:params];
+  BOOL isForward = [[result objectForKey:@"direction"] boolValue];
```
Comment (maxkupetskii) [inline sdk/iOS/OoyalaSkinSDK/React/OOReactSkinBridgeModuleMain.m]: Please, use shorthand syntax, i.e. `result[@"direction"]`

**Review by maxkupetskii — CHANGES_REQUESTED** [review]

File: sdk/iOS/OoyalaSkinSDK/React/OOReactSkinBridgeModuleMain.m (hunk 1 above)
Comment (cacel) [reply sdk/iOS/OoyalaSkinSDK/React/OOReactSkinBridgeModuleMain.m]: Done.

File: sdk/iOS/OoyalaSkinSDK/React/OOReactSkinBridgeModuleMain.m [hunk 2]
```diff
@@ -146,6 +146,16 @@ - (void)setSkinViewDelegate:(nonnull id<OOReactSkinModelDelegate>)delegate {
   }
 }
 
+RCT_EXPORT_METHOD(onSwitch:(NSDictionary *)params) {
+  NSMutableDictionary *result = [[NSMutableDictionary alloc] initWithDictionary:params];
```
Comment (maxkupetskii) [inline sdk/iOS/OoyalaSkinSDK/React/OOReactSkinBridgeModuleMain.m]: Sorry, I’ve missed this part 
Why do we need to create a separate dictionary?
It’s safe to access properties directly in `params` dictionary

Comment (cacel) [reply sdk/iOS/OoyalaSkinSDK/React/OOReactSkinBridgeModuleMain.m]: Done, the dictionary was removed.

**Review by maxkupetskii — APPROVED** [review]

**Pull request merged**